backend/src/services/model.ts

```diff
@@ -319,6 +322,18 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   if (modelDiff.settings?.mirror?.destinationModelId && modelDiff.settings?.mirror?.sourceModelId) {
     throw BadReq('You cannot select both mirror settings simultaneously.')
   }
+  if (modelDiff.collaborators) {
+    const invalidUsers = []
```

Comment:
Can we type this array please? I also thing we should change this to store the `collaboration.entity` so that we can inform users of what has failed by returning exactly what they provided.
```
const invalidUser: CollaboratorEntry[] = []
```